spec/system/provider_interface/removing_provider_user_spec.rb

```diff
@@ -68,6 +69,10 @@ def then_the_deleted_user_has_no_visible_provider_permissions
     expect(@user_to_remove.reload.providers).to eq([@non_visible_provider])
   end
 
+  def and_the_audit_log_has_been_updated
+    expect(@user_to_remove.own_and_associated_audits.first(2).map(&:action)).to eql(%w[destroy destroy])
```

Comment:
Could be more explicit about the type that's being destroyed?

```ruby
@user_to_remove.own_and_associated_audits.limit(2).pluck(:action, :auditable_type).to eql [["destroy", "ProviderPermissions"], ["destroy", "ProviderPermissions"]]
```

Reply:
Done!